src/azure-cli/azure/cli/command_modules/storage/commands.py

```diff
@@ -284,8 +286,6 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
                                     'undeleted'),
                                 table_transformer=transform_boolean_for_table, min_api='2017-07-29')
 
-        g.storage_custom_command_oauth('upload', 'upload_blob',
```

Comment:
>  g.storage_custom_command_oauth('upload', 'upload_blob', [](start = 6, length = 57)

what about Azure stack if you delete this one

Reply:
I keep the original method call of Track 1 in another file `commands_azure_stack.py`.
https://github.com/Azure/azure-cli/blob/4d58aaee1266a46ed320c9cf21e879163f05ba91/src/azure-cli/azure/cli/command_modules/storage/commands_azure_stack.py#L269